README.rst

```diff
@@ -126,8 +126,7 @@ Install geomstats : Developers
 ------------------------------
 
 Developers should git clone the master branch of this repository, together with the development requirements
-and the optional requirements to enable ``tensorflow`` and ``pytorch``
-backends:
+and the optional requirements to enable ``pytorch`` backends:
```

Comment:
Mention autograd backend here (it should have been done before, but it seems we forgot)

Reply:
Done.